sechub-sarif/src/main/java/com/daimler/sechub/sarif/model/ArtifactLocation.java

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.sarif.model;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+/**
+ * Location of an artifact, look at see <a href=
+ * "https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/sarif-v2.1.0-os.html#_Toc34317613">SARIF
+ * 2.1.0 specification entry</a>
+ * 
+ * @author Albert Tregnaghi
+ *
+ */
+@JsonPropertyOrder({ "uri", "uriBaseId" })
+public class ArtifactLocation {
+    private String uri;
+    private String uriBaseId;
+
+    public ArtifactLocation() {
+    }
+
+    public ArtifactLocation(String uriBaseId, String uri) {
+        this.uriBaseId = uriBaseId;
+        this.uri = uri;
+    }
+
+    public String getUri() {
+        return uri;
+    }
+
+    public void setUri(String uri) {
```

Comment:
It's inside the report - when the URI is not valid the tool is in charge. It is just handled as a string. If we do not rely on the tools we would add some additional meta information with the wrong data if not valid etc. 

My opinion: we let it as is, if there is a need in future, this will be another issue.

Reply:
That is a totally valid explanation.